### PR TITLE
(MODULES-7469) Add password_max_age parameter

### PR DESCRIPTION
**Pull request by eimlav**

### CODE REVIEW

File: manifests/user.pp [hunk 1]
```diff
@@ -193,6 +193,7 @@
   Boolean                              $managevim                = true,
   Enum['inclusive','minimum']          $membership               = 'minimum',
   String                               $password                 = '!!',
+  Optional[Integer[0,99999]]           $password_max_age         = undef,
```
Comment (pillarsdotnet) [inline manifests/user.pp]: This should be `Optional[Accounts::User::PasswordMaxAge]    $password_max_age    = undef,`

Comment (eimlav) [reply manifests/user.pp]: Good spot! I have changed this now.

**Comment by eimlav** [comment]
Adhoc ran successfully:

<details><summary>See results here</summary>Screen Shot 2019-05-13 at 13 59 44</details>

**Comment by carabasdaniel** [comment]
LGTM 👍

**Pull request merged**